flake.nix

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
It would probably be wise to not use the unstable flake format for this but to use the "classic" packaging for Nix packages here. This format has not been formalized or finalized.

Reply:
Thanks for reviewing! The NixOS Foundation decided to package Software worked on during NGI Zero subprogram using the Flakes format. The is not obligation to include this upstream.